app/validators/phone_number_validator.rb

```diff
@@ -1,6 +1,8 @@
 class PhoneNumberValidator < ActiveModel::EachValidator
   def validate_each(record, attribute, value)
-    if value.blank? || is_invalid_phone_number_format?(value)
+    if value.blank?
+      record.errors[attribute] << I18n.t('activemodel.errors.models.candidate_interface/contact_details_form.attributes.phone_number.blank')
```

Comment:
`record.errors.add(attribute, :blank)`?
Then Rails will defer to the right message automatically?

Reply:
record.errors.add(attribute, :blank) seems to default to `can't be blank` 🤔 
I'll have a look into it.

Reply:
It's how the Validation tests are set up. We use a `Validatable` model as a stub, which throws the locales off as it's not `candidate_interface/contact_details_form`.